src/software/field_tests/ball_placement_field_test.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+import software.python_bindings as tbots_cpp
+import sys
+import math
+from proto.ssl_gc_common_pb2 import Team
+from proto.ssl_gc_state_pb2 import Command
+from proto.import_all_protos import *
+from software.field_tests.field_test_fixture import *
+
+from software.simulated_tests.simulated_test_fixture import *
+from software.logger.logger import createLogger
+from software.simulated_tests.robot_enters_region import RobotEventuallyEntersRegion
+from proto.message_translation.tbots_protobuf import create_world_state
+
+logger = createLogger(__name__)
+
+
+def test_ball_placement(field_test_runner):
```

Comment:
add docs about what this test does

Reply:
By python convention, the doc string should be below the function declaration:
```suggestion
def test_ball_placement(field_test_runner):
    """"This field test will start a..."""
```